### PR TITLE
UIREC-181 Confirm user would like to delete holdings record when chan…

### PR DESCRIPTION
**Pull request by usavkov-epam**
…ging Holdings link of Piece

## Purpose

https://issues.folio.org/browse/UIREC-181
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/TitleDetails/AddPieceModal/AddPieceModal.js [hunk 1]
```diff
@@ -111,7 +155,7 @@ const AddPieceModal = ({
         data-test-add-piece-save
         disabled={hasValidationErrors}
         marginBottom0
-        onClick={handleSubmit}
+        onClick={(id && initialHoldingId) ? onSave : handleSubmit}
```
Comment (AndreiShumski2) [inline src/TitleDetails/AddPieceModal/AddPieceModal.js:158]: `id && initialHoldingId` condition could be moved to `onSave`

File: src/TitleDetails/TitleDetailsContainer.js [hunk 2]
```diff
@@ -70,6 +70,34 @@ const TitleDetailsContainer = ({ location, history, mutator, match }) => {
...
   );
 
+  const getHoldingsItemsAndPieces = useCallback((holdingId, params = {}) => {
+    const holdingsPieces = mutator.pieces.GET({
+      params: {
+        limit: `${LIMIT_MAX}`,
+        query: `holdingId==${holdingId}`,
+        ...params,
+      },
+      records: undefined,
```
Comment (AndreiShumski2) [inline src/TitleDetails/TitleDetailsContainer.js:80]: is it required?

Comment (usavkov-epam) [reply src/TitleDetails/TitleDetailsContainer.js:80]: If I don't specify the `records` field as `undefined`, then I get only `items` record, which specified in manifest.
In the case with confirm modal I don't need all items records, only their `totalRecords`.

Comment (usavkov-epam) [reply src/TitleDetails/TitleDetailsContainer.js:80]: So, I can set `limit` to `1` and get `items` and `totalRecords` data from the request

**Pull request merged**